site/gatsby-site/src/templates/citeDynamicTemplate.js

```diff
@@ -129,6 +129,9 @@ function CiteDynamicTemplate({
           <Spinner />
         </div>
       )}
+      <div>
+        {String(loading)} : {String(loadingIncident)}
+      </div>
```

Comment:
I don't understand this code. It looks like it's a debugging code since `loading` and `loadingIncident` are boolean variables

Reply:
Woor that should have never been pushed